docs/tasks/administer-cluster/cloud-controller-manager-deployment-example.yaml

```diff
@@ -0,0 +1,59 @@
+# This is an example of how to setup cloud-controller-manager as a Deployment in your cluster.
```

Comment:
I'm not sure we need two examples... it's straightforward to do the conversion from DS to Deployment IAC, but high maintenance cost

Reply:
ping, can you remove this?